docs/data/material/guides/pickers-migration/pickers-migration.md

```diff
@@ -123,13 +133,17 @@ Mask is no longer required. Also, if your provided mask is not valid, pickers wi
 
 ## And many more
 
-- ```diff
-   <DatePicker
-  -  format="DD-MMM-YYYY"
-  +  inputFormat="DD-MMM-YYYY"
-  ```
+```diff
+ <DatePicker
+-  format="DD-MM-YYYY"
++  inputFormat="DD-MM-YYYY"
```

Comment:
Also applied a change to avoid possible issue discussed in: https://mui-org.slack.com/archives/C041SDSF32L/p1673608071414259